### ISSUE TITLE
oce.axis.POSIXct `xaxp` error

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1094]
When I make a time series plot using `oce.plot.ts()` (other other things that use `oce.axis.POSIXct()` like `imagep()`) with a time range between 5.5 hours and 11 hours, it fails with the error and traceback:
```
Error in par(xaxp = c(min(zzz, na.rm = TRUE), max(zzz, na.rm = TRUE),  : 
  invalid value specified for graphical parameter "xaxp" 
7. par(xaxp = c(min(zzz, na.rm = TRUE), max(zzz, na.rm = TRUE), 
    -1 + length(zzz))) at oce.R#2436
6. oce.axis.POSIXct(1, x = x, drawTimeRange = drawTimeRange, main = main, 
    mgp = mgp, xlim = if (missing(xlim)) range(x) else xlim, 
    cex = cex, cex.axis = cex.axis, cex.main = cex.main, tformat = tformat, 
    debug = debug - 1) at oce.R#944
5. oce.plot.ts(t, yy) at a.R#10
4. eval(expr, envir, enclos) 
3. eval(ei, envir) 
2. withVisible(eval(ei, envir)) 
1. source("~/src/oce-issues/tmp/a.R") 
```

The problem seems to be happening because the decision tree that decides how many labels to place only puts one (which isn't ideal, obviously), and then the calculation of `xaxp` gets mucked up.

**Comment by richardsc** [comment]
Example code can be found in `oce-issues/1094`

**Comment by dankelley** [comment]
I think this is fixed in `develop` commit 7f81a01704c09e555f675cc8e630f8fbb99ea0de

**Comment by richardsc** [comment]
Great, thanks!
